app/static/src/app/components/sessions/SessionsPage.vue

```diff
@@ -4,20 +4,58 @@
       <errors-alert></errors-alert>
       <h2>Sessions</h2>
     </div>
-    <template v-if="sessionsMetadata">
-      <template v-if="sessionsMetadata.length">
-        <div class="row fw-bold py-2">
+    <div class="row mb-3" id="no-current-session" v-if="!currentSession">
```

Comment:
If you are quick going from the app to the session page, it's possible to land here before the current session is saved, and then there's no "return to current session" link which is surprising.